python/spglib/spglib.py

```diff
@@ -40,7 +40,7 @@
 import numpy as np
 
 try:
-    from spglib import _spglib as spg
+    from spglib import _spglib as spg  # type: ignore
```

Comment:
Why the ignore? Also should scope ignores

Reply:
Without ignoring, mypy says
```
error: Module "spglib" has no attribute "_spglib"  [attr-defined]
```

Reply:
Yeah, but that's a genuine issue:
- We should be using relative paths there
- We should be providing a `_spglib.pyi` to type-hint the C interface

Reply:
The relative path did not persuade mypy
```
    from .spglib import _spglib as spg
# -> Module "spglib.spglib" has no attribute "_spglib"  [attr-defined]
```